### PR TITLE
Changes to provide custom wooey site name branding

### PR DESCRIPTION
**Pull request by hottwaj**
WOOEY_SITE_NAME and WOOEY_SITE_TAG added to settings
wooey_settings added to context of some views so WOOEY_SITE_NAME and WOOEY_SITE_TAG can be picked up in templates

todo:
some other views still need tweaking - maybe use a Mixin to centralise passing of site settings?
wooey base app url should also be configurable? (or is this a django urls.py issue?)
set html "title"

### CODE REVIEW

File: wooey/settings.py [hunk 1]
```diff
@@ -15,4 +15,6 @@ def get(key, default):
 WOOEY_REGISTER_URL = get('WOOEY_REGISTER_URL', '/accounts/register/')
 WOOEY_SHOW_LOCKED_SCRIPTS = get('WOOEY_SHOW_LOCKED_SCRIPTS', True)
 WOOEY_EPHEMERAL_FILES = get('WOOEY_EPHEMERAL_FILES', False)
-WOOEY_DEFAULT_SCRIPT_GROUP = get('WOOEY_DEFAULT_SCRIPT_GROUP', _('Wooey Scripts'))
\ No newline at end of file
+WOOEY_DEFAULT_SCRIPT_GROUP = get('WOOEY_DEFAULT_SCRIPT_GROUP', _('Wooey Scripts'))
```
Comment (mfitzp) [inline wooey/settings.py]: I wonder if we should change the default script group name to "Scripts" to remove the use of the name in the admin.

**Comment by mfitzp** [comment]
Thanks for this @hottwaj much appreciated.

As in the comment about I wonder if we should drop the "Wooey" from the default script group name @Chris7?

Adding the `WOOEY_SITE_TAG` and `WOOEY_SITE_NAME` to the context would probably be better done using a context processor. That way we don't need to add the passing code to every view. Here is an [example of how to do it]. I'd suggest adding a file named `context_processors.py` in the root of the Wooey app folder.

I'm not sure if it's a good idea to pass the whole `SETTINGS` var into the template context since it contains the database passwords etc. Instead you can pass the two names as-is above.

**Comment by Chris7** [comment]
The issue is for pluggable apps, you'd require them to install our middleware/context_processors as well. I'll see if there's an easier way to avoid this. One approach is using template tags to fetch from our settings like {% get_wooey_setting xxx %}

File: wooey/settings.py [hunk 2]
```diff
@@ -15,4 +15,6 @@ def get(key, default):
...
+WOOEY_SITE_NAME = get('WOOEY_SITE_NAME', 'Wooey!')
+WOOEY_SITE_TAG = get('WOOEY_SITE_TAG', 'A web UI for Python scripts')
```
Comment (Chris7) [inline wooey/settings.py]: Can you wrap these in the translation tag as well?

**Comment by mfitzp** [comment]
Ah cool. Perhaps even simpler `{% wooey_site_name %}` `{% wooey_site_tagline %}` unless we're going to want to pull through multiple settings?

I'm a bit wary of making the settings file accessible through the template though I'm not entirely sure why.

**Comment by Chris7** [comment]
I think that would lead to a lot of redundant template tags. This wouldn't make database settings accessible, and if we ever do have sensitive things in the settings, we can whitelist them before fetching it as well.

def wooey_setting(val):
 whitelist = {a,b,c}
 return wooey_settings.get(val) if val in whitelist else None

**Comment by mfitzp** [comment]
:+1: was confusing the Wooey settings with the main ones. @hottwaj Take a look here: http://stackoverflow.com/a/7716141/754456 for an example implementation.

**Comment by hottwaj** [comment]
All sounds good - I like the template tag idea.  I'll make those changes and upload...

**Comment by hottwaj** [comment]
Sure, see test attached - that what you had in mind?

**Comment by Chris7** [comment]
Thanks @hottwaj. :+1: from me, Martin?

**Comment by mfitzp** [comment]
Great stuff @hottwaj nice one!

**Comment by hottwaj** [comment]
No problem thanks guys

**Pull request merged**